app/src/org/commcare/tasks/SendTask.java

```diff
@@ -91,6 +94,26 @@ protected Boolean doTaskBackground(Void... params) {
                 continue;
             }
             try {
+
+                // see if we have a form.properties file to load the PostURL from
+                FilenameFilter filter = new FilenameFilter() {
+                    @Override
+                    public boolean accept(File dir, String filename) {
+                        return filename.equals(ZipTask.FORM_PROPERTIES_FILE);
+                    }
+                };
+
+                File[] fileContents = f.listFiles(filter);
+                if(fileContents != null && fileContents.length > 0){
+                    Properties properties = FileUtil.loadProperties(fileContents[0]);
+                    if(properties != null && properties.getProperty(ZipTask.FORM_PROPERTY_POST_URL) != null){
+                        url = properties.getProperty(ZipTask.FORM_PROPERTY_POST_URL);
+                        Logger.log(TAG, "Successfully got form.property PostURL: " + url);

```

Comment:
This will send the log to HQ with the tag `SendTask`, which is the only time such a tag is sent to HQ. Consider using `AndroidLogger.TYPE_FORM_DUMP` to match other wifi/sd form dump logging calls.